src/NServiceBus.Core.Tests/Serializers/XML/Issue_2796.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Core.Tests.Serializers.XML
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Serializers.XML.Test;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class Issue_2796
```

Comment:
Do we really want to name our test classes after issues? I realise there are precedents, but does it make sense?